misc/toml-parser/src/main/java/io/ballerina/toml/internal/parser/TomlParser.java

```diff
@@ -191,27 +202,35 @@ private STNode parseKeyValue() {
         STNode identifier = parseKeys();
         STNode equals = parseEquals();
         STNode value = parseValue();
+        STNode newLines = parseNewLines();
```

Comment:
Parser should use newline token(s) only to make parsing-decisions. It should not use these tokens when creating nodes. Newlines should come from the minutiae as before.

Reply:
Let's discuss here,
https://github.com/ballerina-platform/ballerina-lang/pull/26429#issuecomment-720410806